langspec/xproc11/xproc.xml

```diff
@@ -1027,7 +1030,7 @@ refer to that variable will raise an error.</para>
               </listitem>
             </varlistentry>
             <varlistentry>
-              <term>Base URI</term>
+              <term>Static base URI</term>
               <listitem>
                 <para>The base URI of the element on which the expression occurs.</para>
```

Comment:
I don't think this is the static base URI; I think this is the "dynamic" base URI of the actual element in the document.